odoo/modules/loading.py

```diff
@@ -61,9 +62,10 @@ def _get_files_of_kind(kind):
         return files
 
     filename = None
+    is_testing = module.current_test
     try:
         if kind in ('demo', 'test'):
-            threading.current_thread().testing = True
+            module.current_test = True
```

Comment:
This is not thread-safe.

Reply:
Solved by not setting anymore "testing" on the thread while loading demo data (test data is never loaded using this function, but directly created in tests). Besides, loading demo is not testing.